### ISSUE TITLE
User registration broken

### ISSUE THREAD

**Issue by slinkp** [openplans/openblock#281]
- Enter email address to register
- Click confirmation link
- Submit password entry form. Kaboom.

```

AttributeError at /accounts/c/

'User' object has no attribute 'backend'

Traceback:
File "/var/www/demo.openblockproject.org/current/lib/python2.6/site-packages/django/core/handlers/base.py" in get_response
  111.                         response = callback(request, *callback_args, **callback_kwargs)
File "/var/www/demo.openblockproject.org/builds/20110815/src/openblock/ebpub/ebpub/accounts/views.py" in confirm_email
  237.             utils.login(request, u)
File "/var/www/demo.openblockproject.org/builds/20110815/src/openblock/ebpub/ebpub/accounts/utils.py" in login
  89.     return auth.login(request, user)
File "/var/www/demo.openblockproject.org/current/lib/python2.6/site-packages/django/contrib/auth/__init__.py" in login
  82.     request.session[BACKEND_SESSION_KEY] = user.backend


```

**Comment by slinkp** [comment]
(In [caf391e8eca351be3d7e1f4a1150be7be9a0cee0]) Set user.backend on login. Fixes #288

**Comment by slinkp** [comment]
Ticket imported from Trac:
 http://developer.openblockproject.org/ticket/288
Reported by: slinkp
